### PR TITLE
Fix nova request plugin

### PR DESCRIPTION
**Pull request by Taure**
Fixed issue in nova_routes where we added plugins to Env variable instead of cowboy req.

### CODE REVIEW

File: src/nova_cors_plugin.erl [hunk 1]
```diff
@@ -2,24 +2,23 @@
...
          plugin_info/0]).
 
-pre_http_request(#{req := Req} = NovaState, _) ->
+pre_request(Req, _) ->
     Req1 = cowboy_req:set_resp_header(<<"Access-Control-Allow-Origin">>, <<"*">>, Req),
     Req2 = cowboy_req:set_resp_header(<<"Access-Control-Allow-Headers">>, <<"*">>, Req1),
     Req3 = cowboy_req:set_resp_header(<<"Access-Control-Allow-Methods">>, <<"*">>, Req2),
-    NewState = maps:put(req, Req3, NovaState),
-    case Req of
+    case Req3 of
```
Comment (burbas) [inline src/nova_cors_plugin.erl:13]: Wouldn't it make more sense to pattern match against the <<"OPTIONS">> method instead?

**Review by burbas — DISMISSED** [review]
Fix the pattern match in nova_cors_plugin and I'm a happy panda 👍

File: src/nova_cors_plugin.erl (hunk 1 above)
Comment (Taure) [reply src/nova_cors_plugin.erl:13]: Doesn't we need the cors even on OPTIONS? And patter match where on what?

Comment (burbas) [reply src/nova_cors_plugin.erl:13]: I'm not 100% sure tbh if we need it for OPTIONS. But now I just realized that my idea about pattern patching was wrong. Sorry about that!

**Pull request closed without merging**